modules/uploading-model-files-to-pvc.adoc

```diff
@@ -0,0 +1,41 @@
+:_module-type: PROCEDURE
+
+[id="uploading-model-files-to-pvc_{context}"]
+
+= Uploading model files to a Persistent Volume Claim (PVC)
+
+When deploying a model, you can serve it from a preexisting Persistent Volume Claim (PVC) where your model files are stored. You can upload your local model files to a PVC by using a running Jupyter notebook workbench.
```

Comment:
I find "jupyter notebook workbench" confusing. In our RHOAI  doc, we don't refer to workbenches as Jupyter notebook workbenches - just workbenches or project workbenches

Reply:
also - does this mean they cannot upload to a PVC if the reader's IDE is code-server (not Jupyter)